uptime-kuma/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       APP_PORT: 3001
 
   uptime-kuma:
-    image: louislam/uptime-kuma:1.16.1@sha256:f5706f21ba0af08775505aeddf61aff1d344038f6df5a90865699b86677dfd99
+    image: louislam/uptime-kuma:1.17.1@sha256:ee729a48ff4a7a8485fbe6cf30a0bc63cdf49f74c541d73e301334633f1c31c2
```

Comment:
Just FYI, the digest here is for amd64. We requre a multi-arch digest which in this case is `sha256:a4eab252e5a27ead9e92f04d755d6cc968be923b277aae42ac8cf633e2da3b15`

You can get using `docker pull`:
```
root@ubuntu:~# docker pull louislam/uptime-kuma:1.17.1
1.17.1: Pulling from louislam/uptime-kuma
Digest: sha256:a4eab252e5a27ead9e92f04d755d6cc968be923b277aae42ac8cf633e2da3b15
Status: Image is up to date for louislam/uptime-kuma:1.17.1
docker.io/louislam/uptime-kuma:1.17.1
```